packages/solidjs/rollup.npm.config.mjs

```diff
@@ -1,3 +1,9 @@
 import { makeBaseNPMConfig, makeNPMConfigVariants } from '@sentry-internal/rollup-utils';
 
-export default makeNPMConfigVariants(makeBaseNPMConfig());
+export default makeNPMConfigVariants(
+  makeBaseNPMConfig({
+    packageSpecificConfig: {
+      external: ['solid-js', 'solid-js/web', '@solidjs/router'],
```

Comment:
hmm, is this needed? shouldn't this be automatically marked as external? 🤔

Reply:
I think it might not be needed. I didn't have `solid-js` in there before and everything built and worked fine. Will try to remove.

Reply:
Confirmed, don't need. Removed it.

Reply:
I am pretty confident `solid-js/web` is still needed because it is not added to `external` because it is not inside any dependencies.

Reply:
It's not in the build output, I double-checked again.